developers/pymavlink/manual_control.py

```diff
@@ -0,0 +1,40 @@
+"""
+Example of how to send MANUAL_CONTROL messages to the autopilot using
+pymavlink.
+This message is able to fully replace the joystick inputs.
+"""
+# disable warning for "buttons" not being a constant(UPPERCASE)
+# pylint: disable=C0103
```

Comment:
Can we just fix that and not use a bunch of pylints macros everywhere ?

Reply:
We could, but I don't think we should. Making them uppercase will likely confuse users into thinking that they should not be changed.

Reply:
While they are only constant because we only gave one example and it wasn't changed

Reply:
The users will also be confuse about this pylint disable magic codes, I believe that's better to do the correct thing in Python and fix the pylint warnings.

Reply:
You could also disable the warning globally via the command line, no ?

Reply:
The users will also be confuse about this pylint disable magic codes

This could confuse them, but not lead them to write bad code

    You could also disable the warning globally via the command line, no ?

I could, but them I would be enabling us to do more wrong stuff in the future.

I could enable only for C0103 (the uppercase thing), though. I think that would be good.

The other relevant case I remember is the "broad exceptions" warning, that I couldn't fix because I have no idea what were the exception being caught there (I found none in my tests and code inspection)

Reply:
Looks reasonable